tests/management/commands/test_graph_models.py

```diff
@@ -27,3 +27,27 @@ def test_graph_models_json():
     assert output.endswith("]}\n")
     assert """"app_name": "django.contrib.auth", """ in output
     assert "created_at" in json.loads(output)
+
+
+def test_disable_abstract_fields_not_active():
+    out = StringIO()
+    call_command('graph_models',
+                 app_label=['django_extensions'],
+                 include_models=['AbstractInheritanceTestModelChild'],
+                 disable_abstract_fields=False,
+                 stdout=out)
+    output = out.getvalue()
+    print(output)
```

Comment:
please remove the debug print statement :-)

Reply:
Oups, I went to fast sorry

Reply:
It's corrected